contrib/utilities/indent

```diff
@@ -44,3 +44,23 @@ fi
 # with up to 10 in parallel
 find tests include source examples \( -name '*.cc' -o -name '*.h' \) -print | xargs -n 50 -P 10 astyle --options=contrib/utilities/astyle.rc
 
+# format .inst.in files. We need to replace \{ and \} because it confuses
+# astyle.
+format_inst()
+{
+    f=$1
+    cp $f $f.tmp
+    sed -i 's#\\{#{ //#g' $f.tmp
+    sed -i 's#\\}#} //#g' $f.tmp
+    astyle --quiet $f.tmp
```

Comment:
It probably doesn't matter at all, but you're calling `astyle` with the default settings, whereas we have our own style file. For consistency, it may be worthwhile using our own style file here as well.